flake-info/src/data/export.rs

```diff
@@ -230,8 +241,7 @@ impl From<import::NixpkgsEntry> for Derivation {
 
                 let package_maintainers_set = package_maintainers
                     .iter()
-                    .filter(|m| m.name.is_some())
-                    .map(|m| m.name.to_owned().unwrap())
+                    .flat_map(|m| m.name.to_owned())
```

Comment:
Oh `flat_map` unboxes `Options`? Thats neat

Reply:
Yeah, both Option and Result are iterators over their (possibly absent) success value